soh/soh/Enhancements/randomizer/item.cpp

```diff
@@ -100,7 +67,7 @@ ItemType Item::GetItemType() const {
     return type;
 }
 
-std::variant<bool*, uint8_t*> Item::GetLogicVar() const {
+LogicVal Item::GetLogicVar() const {
```

Comment:
Due to the change, there's now `LogicVal` & `LogicVar`, probably a good idea to rename one to the other.

Reply:
Good catch.